### DISCUSSION TITLE
ViewField showing issue on VSC extension

### DISCUSSION THREAD

**Discussion by dharen008** [filamentphp/filament#12882] [Help]
### Package

Panel builder

### Package Version

3

### How can we help you?

Using this code https://filamentphp.com/docs/3.x/forms/fields/custom#view-fields

VSC PHP extensions by Devsense showing a error/problem
Argument '1' passed to view() is expected to be of type Filament\Support\Components\view-string|null, string givenPHP(PHP0406)

even there is a problem showing up, the code still work.

**Comment by BoomCheeseBurge** [comment]
Apparently, I have the same issue. I believe this issue is similar to how the PHP Intelephense extension in VSCode considers the method 'user()' to be an undefined method in the method chaining 'auth()->user()->id' from Laravel. I prefer to ignore it as the code itself works.
